c2cgeoform/static/src/interactions.js

```diff
@@ -9,28 +10,35 @@ const format = new GeoJSON()
 
 export function addInteractions(options) {
   const { input, multi, ...interactionOptions } = options
+  const store = geometry => {
+    input.value = geometry ? getGeometryJSON(geometry, multi) : ''
+  }
   const draw = addDrawInteraction(interactionOptions)
   const modify = addModifyInteraction(interactionOptions)
   modify.setActive(false)
-  options.source.on(
-    'change',
-    e =>
-      (input.value =
-        options.source.getFeatures().length > 0
-          ? getGeometryJSON(
-              options.source.getFeatures()[0].getGeometry(),
-              multi
-            )
-          : '')
+  options.source.on('change', () =>
+    store(options.source.getFeatures()[0]?.getGeometry())
   )
+  // Mobile drawing
+  if (
+    window.matchMedia('(max-width: 576px)').matches &&
+    interactionOptions.type === 'Point'
+  ) {
+    let map = options.map
+    draw.setActive(false)
+    modify.setActive(false)
+    map.getTargetElement().classList.add('cross')
+    map.updateSize()
+    map.on('moveend', () => store(new Point(map.getView().getCenter())))
```

Comment:
Note that delete button seems useless on mobile with type "Point".

Reply:
It may be if the field is not mandatory. It is hidden in getitfixed